src/locales/scripts/get-validated-locales.js

```diff
@@ -24,6 +24,7 @@ const getValidatedLocales = async () => {
   const allLocales = Object.values(updatedLocaleList).map((locale) => ({
     code: locale.slug,
     name: locale.name,
+    nativeName: locale.nativeName,
```

Comment:
```suggestion
    nativeName: locale.nativeName || locale.name,
```

Reply:
This would probably fix the issue I mentioned in the other comment: all locales will have the `nativeName` (or a fallback), and we will not have to calculate it at runtime.

Reply:
Implemented in dda4d2f.